src/routes/Tool/Tool.tsx

```diff
@@ -290,20 +284,27 @@ const Tool = () => {
         }
       />
 
+      <div className="mb-8 mt-5">
+        <Button
+          onClick={() => void updateEhdotukset()}
+          label={t('tool.update-job-opportunities-list')}
+          variant="accent"
+          disabled={ehdotuksetLoading}
```

Comment:
When search starts and button goes disabled it blends little too much to the background.

_It seems that there is already some question in design about the Button._

Reply:
Yeah, this is a MVP implementation at the moment. Let PAMU figure out a better way.